### ISSUE TITLE
Confusing message after signing up

### ISSUE THREAD

**Issue by Raxerz** [systers/macc#195]
![screenshot from 2017-03-02 23 49 15](https://cloud.githubusercontent.com/assets/16686007/23520960/0c64398e-ffa3-11e6-8208-bc88992e6c1c.png)

The message currently reads "Go Back or click OK to go to signup page".
There is no OK button that can be clicked on.

**Comment by muddlebee** [comment]
@Raxerz put up a PR for the same. @medhach take a look please

**Comment by Raxerz** [comment]
@anweshknayak  I can submit a PR but it'll help if there is a specific or a preferred flow for the signing up process.
For example, Should there be an OK button to take the user to the sign up page or should it just be a "Thank you for signing up" page with the "verification mail sent" message allowing the user to access his account only after the verification is done.
Also, redirecting the user to the "sign up" page after signing up seems to be redundant.
@anweshknayak @medhach Please let me know if you have any other suggestions regarding this.

**Comment by medhach** [comment]
@Raxerz If you can make the user get his verification link and verify his account, well and good. If you want to add a Thank you for signing up, you can do that too but make sure you create another issue for verifying emails. :)

**Comment by Raxerz** [comment]
Alright @medhach. Will start working on it.
